tests/common.py

```diff
@@ -11,7 +11,8 @@
 
 SCHEMAS_PATH = RESOURCES_PATH / "schemas" / "stsci.edu" / "asdf"
 DOCS_PATH = ROOT_PATH / "docs" / "source"
-DOCS_SCHEMAS_PATH = DOCS_PATH / "schemas"
+DOCS_SCHEMAS_PATH = DOCS_PATH
+DOCS_SCHEMAS_LIST = ["asdf.rst", "astronomy.rst", "core.rst", "legacy.rst"]
```

Comment:
What's the need for moving these out of a "schemas" subdirectory? I'm in favor of keeping to old structure if it doesn't cause complications (since it's easier to tell what portions of the docs are describing schema resources).